cirq/google/sim/xmon_simulator.py

```diff
@@ -166,6 +167,35 @@ def __init__(self, options: XmonOptions = None) -> None:
         """
         self.options = options or XmonOptions()
 
+    def wavefunction(self, state, decimals=2):
+        """
+        Returns the wavefunction as a string in Dirac notation.
+
+        For example:
+        circuit = cirq.Circuit.from_ops(cirq.H(Q1))
+        simulator = cirq.google.XmonSimulator()
+        result = simulator.simulate(circuit)
+
+        print(wavefunction(result.final_state)) -> -0.71j|0> + -0.71j|1>
```

Comment:
Does H really produce complex values here (i.e the j)?

Reply:
Yes. It's kind of strange, but `abs(0.71j)^2` does get you `~0.5`.

Reply:
It's because H is being decomposed into exp(i Y pi/2) * exp(i Z pi).